packable/packable/CHANGELOG.md

```diff
@@ -22,6 +22,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## 0.X.X - 2022-XX-XX
 
+### Added
+
+- Implement `Packable` for tuples;
```

Comment:
Anything else planned soon or should we do a version for it ?

Reply:
yeah I've planned to do the `(un)?pack_with` thingy

Reply:
What's that again ?